mackup/applications/storyist-3.cfg

```diff
@@ -0,0 +1,6 @@
+[application]
+name = storyist-3
+
+[configuration_files]
+Library/Preferences/.com.storyist.storyist.plist
```

Comment:
This file really exists? Never saw a hidden plist file.

Reply:
Yes. It's a Storyist specific thing. If purchased, your "key" is in there. Otherwise it's a garbage key which lets Storyist know it's in trial mode. A non-standard way to do things for sure.

Reply:
👌🏼